diesel/src/query_builder/mod.rs

```diff
@@ -55,7 +55,7 @@ pub use self::sql_query::{BoxedSqlQuery, SqlQuery};
 pub use self::upsert::on_conflict_target_decorations::DecoratableTarget;
 
 #[doc(inline)]
-pub use self::update_statement::changeset::AsChangeset;
+pub use self::update_statement::changeset::{AsChangeset, AssignmentTarget};
```

Comment:
I would prefer to not expose `AssignmentTarget` as part of the public API yet. Can we use a crate local reexport instead?